Screens/ImageCapture.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native'
+import Camera from 'react-native-camera'
+
+class ImageCapture extends Component {
+  state = {
+    images: []
+  }
+
+  static navigationOptions = {
+    title: 'Camera'
+  }
+
+  takePicture() {
+    const options = {}
+    this.camera.capture({ metadata: options })
```

Comment:
>{ metadata: options } [](start = 24, length = 21)

Doesn't look like you're actually sending any options. You can probably delete this.